doc/doc-site/docs/off-chain/data-service-stack.md

```diff
@@ -1,39 +1,44 @@
 # Data service stack
 
 The Econia data service stack (DSS) is a collection of services that provide assorted data endpoints for integration purposes.
-It exposes a REST API and a WebSocket server, and is composed of multiple services (database, API server, WebSocket server, indexer, …).
-To make it easy to run, we have dockerized each service, and have created a Docker compose file to run everything easily.
-On this page, we're going to see how to set everything up.
+It exposes a REST API and a WebSocket server, which are powered internally by an aggregator, database, and indexer.
```

Comment:
Shouldn't these be:

> by an aggregator, a database, and an indexer.

Since there is an "an" before "aggregator", it seems to me that there should be an "a" before "database" and an "an" before "indexer".

Reply:
Resolved as of `e43da247` (document relocated due to doc tree re-shuffle)